docs/topics/rules-important-links.adoc

```diff
@@ -6,6 +6,6 @@
 [id="rules-important-links_{context}"]
 = Additional resources
 
-* {ProductShortName} Javadoc: http://windup.github.io/windup/docs/latest/javadoc
+// * {ProductShortName} Javadoc: http://windup.github.io/windup/docs/latest/javadoc
```

Comment:
As this is used in line 108, lets use it here as well

```suggestion
 * {ProductShortName} Javadoc: https://github.com/windup/windup/blob/master/reporting/api/src/main/java
```

Reply:
@anarnold97 The link to Javadocs (the current display text) is broken, so I think we should comment this out for now. I've alerted the windup team about the problem. Once they give us the correct link, we can add it.

Reply:
Please address this comment

Reply:
I don't understand the question. Line 108 of document-attributes.adoc previously (MTA 6.0 and earlier)  contained an attribute named :LinkAPI: that was defined as : http://windup.github.io/windup/docs/latest/javadoc/ . That definition is no longer correct and led to all the broken links. You found the correct path for two of the links (ClassificationExists and HintExists) and we defined LinkAPI based on that. 

When I tried to use that attribute with IterableFilter, I saw it did not work, because IterableFilter was in a different subdirectory than the other two. I chickened out and used the full directory path, instead of truncating :LinkAPI;

You corrected me on that , so I changed  :LinkAPI: and adjusted the links for those 3 items. I checked the links and they work.

You told me previously to comment out line 8 of rules-important-links after I told you the link on the website is broken. 

![image](https://github.com/windup/windup-documentation/assets/60698649/4f5a725c-69f8-49ff-9e67-682e30e17758)

I've searched the repo we cloned and don't see an entry for any spelling or capitalization of Javadocs/Javadox. 

I've let the windup team know the link on the site is broken. 

The change to line 108 of document-attributes.adoc fixes all the links except for the link in the Appendix, line 8 of rules-important-links.adoc. I can't find anything that replaces the link in the Appendix for sure, so I suggest commenting out line 8 of rules-important-links.adoc. I did change the line to include :LinkAPI: .